builtin/logical/pki/path_issue_sign.go

```diff
@@ -179,44 +184,41 @@ func (b *backend) pathIssueSignCert(
 
 	switch format {
 	case "pem":
-		resp.Data["issuing_ca"] = cb.IssuingCA
-		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
+		resp.Data["issuing_ca"] = signingCB.Certificate
 		resp.Data["certificate"] = cb.Certificate
-
 		if !useCSR {
 			resp.Data["private_key"] = cb.PrivateKey
 			resp.Data["private_key_type"] = cb.PrivateKeyType
 		}
 
 	case "pem_bundle":
-		resp.Data["issuing_ca"] = cb.IssuingCA
-		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
+		resp.Data["issuing_ca"] = signingCB.Certificate
 		resp.Data["certificate"] = cb.ToPEMBundle()
+		if cb.CAChain != nil {
```

Comment:
I know this is annoying, but please check for both nil and for length zero. Every now and then this really does catch subtle bugs where a slice was instantiated but not populated with what you expected.